rtv/__main__.py

```diff
@@ -68,6 +69,13 @@ def main():
     args = command_line()
     local_config = load_config()
 
+    # set the terminal title
+    title = 'rtv {0}'.format(__version__)
+    try:
+        sys.stdout.write("\x1b]2;{0}\x07".format(title))
```

Comment:
I found it on the [arch wiki](https://bbs.archlinux.org/viewtopic.php?id=85567).

Windows method (and linux again) on this [stackoverflow post](http://stackoverflow.com/questions/2330393/how-to-set-the-program-title-in-python#answer-2330596)